src/components/SliderWithInput.tsx

```diff
@@ -0,0 +1,128 @@
+import { css } from "@emotion/react";
+import {
+  Input,
+  InputProps,
+  SliderUnstyled,
+  SliderUnstyledProps,
+  Theme,
+} from "@mui/material";
+
+import React from "react";
+
+export type SliderWithInputProps = {
+  label: string;
+  inputProps?: InputProps;
+  sliderProps?: SliderUnstyledProps;
+};
+const SliderWithInput: React.VFC<SliderWithInputProps> = (props) => {
+  const { label, inputProps, sliderProps, ...rest } = props;
+
+  return (
+    <div className="slider-container" css={styles} {...rest}>
+      <label htmlFor="slider-input">{label}</label>
+      <Input id="slider-input" className="slider-input" {...inputProps} />
```

Comment:
We'll need to be careful here in case there's more than one slider on the page, as it'll make the `id` non-unique.

Reply:
Added a unique identifier